src/gov/nasa/worldwind/formats/nitfs/NITFSRuntimeException.java

```diff
@@ -16,43 +14,31 @@
 {
     public NITFSRuntimeException()
     {
-        super();
-    }
-
-    public NITFSRuntimeException(String messageID)
-    {
-        super(Logging.getMessage(messageID));
-        log(this.getMessage());
+        super((String) null);
+        log();
```

Comment:
This call to log is an addition to this constructor. I think we want this removed.